.github/workflows/check-file-updates.yaml

```diff
@@ -0,0 +1,43 @@
+name: Check config and readme updates
+on:
+  pull_request:
+    paths:
+      - "apis/**"
+jobs:
+  file-updates:
+    permissions:
+      pull-requests: write
+    name: Ensure generated files are included
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Generate files
+        run: |
+          make generate manifests api-docs
+      - name: Ensure generated files are up-to-date
+        id: check_generated_files
+        run : |
+          if [[ -n $(git status -s) ]]
+          then
+            echo "Generated files have been missed in the PR"
+            echo "missing_generated_files=true" >> $GITHUB_OUTPUT
+          else
+            echo "No new files to commit"
+            echo "missing_generated_files=false" >> $GITHUB_OUTPUT
+          fi
+      - name: Report issue in PR
+        if: ${{ steps.check_generated_files.outputs.missing_generated_files == 'true' }}
+        uses: thollander/actions-comment-pull-request@v2
+        with:
+          message: |
+              ## This PR can't be merged just yet 😢
+
+              Hey there! It looks like some files aren't up to date. No worries though! Just run the commands below to automatically generate those files and make sure to commit them afterwards:
```

Comment:
do we need so much words? 
or just " run command 'make generate manifests api-docs' and commit changes:"

Reply:
maybe we can add some` Please`, Please run `make generate manifests api-docs` and commit the changes

Reply:
Yeah, lets make it one sentence then.

Reply:
done